Library/src/Systems/Quota/Quota.cpp

```diff
@@ -138,6 +202,38 @@ void UserTierResult::OnResponse(const csp::services::ApiResponseBase* ApiRespons
 	}
 }
 
+int UserTierResult::ParseErrorCode(const csp::common::String& Value)
+{
+	if (Value == "group_spaceownerquota")
```

Comment:
Our [programming guidelines](https://github.com/magnopus-opensource/connected-spaces-platform/wiki/Programming-guidelines) closely follow the Epic coding guidelines, which ask for us to always include braces (see the **Braces** section [here](https://docs.unrealengine.com/5.2/en-US/epic-cplusplus-coding-standard-for-unreal-engine/)

Reply:
sorted